app/assets/javascripts/hyrax/nav_safety.js

```diff
@@ -4,23 +4,40 @@
 // - Add the nav-safety class to the form element.
 
 Blacklight.onLoad(function() {
-  var clickedTab;
-  $('.nav-safety-confirm').on('click', function(evt) {
-    clickedTab = $(this).attr('href');
+  $('.nav-safety-confirm').on('show.bs.tab', function(evt) {
+    clickedTab = $(evt.target).attr('href');
```

Comment:
@bkeese This looks good :+1: I have just one request. Can this line be removed since it is no longer being used?

Reply:
Good catch. Removed.